spec/unit/pops/lookup/interpolation_spec.rb

```diff
@@ -161,6 +161,10 @@ def expect_lookup(*keys)
       expect(interpolator.interpolate('a dot e: %{a.d}', lookup_invocation, true)).to eq('a dot e: (scope) a dot d is a hash entry')
     end
 
+    it 'should report a key missing and replace with empty string when a dotted key is used to navigate into a structure and then not found' do
+      expect(interpolator.interpolate('a dot n: %{a.n}', lookup_invocation, true)).to eq('a dot n: ')
+    end
```

Comment:
alright... kind of hard to parse and understand, but I get it now that I read it again. (merging).